Sources/SourceKitLSP/Swift/SwiftLanguageService.swift

```diff
@@ -554,10 +554,11 @@ extension SwiftLanguageService {
 
     let concurrentEdits = ConcurrentEdits(
       fromSequential: edits.map {
-        IncrementalEdit(
-          offset: $0.range.lowerBound.utf8Offset,
-          length: $0.length.utf8Length,
-          replacementLength: $0.replacement.utf8.count
+        SourceEdit(
+          range: AbsolutePosition(
+            utf8Offset: $0.range.lowerBound.utf8Offset
+          )..<AbsolutePosition(utf8Offset: $0.range.lowerBound.utf8Offset + $0.range.length.utf8Length),
```

Comment:
I think `edits` already is `[SourceEdit]`, so the entire `map` can removed, I think.